.github/workflows/tests.yml

```diff
@@ -0,0 +1,52 @@
+name: tests
+
+on:
+  push:
+    branches:
+      - master
+      - '*.x'
+  pull_request:
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
This will run your build one time during the day to spot regressions in third party libraries.